src/System.Memory/ref/System.Memory.cs

```diff
@@ -246,6 +270,46 @@ public abstract partial class OwnedMemory<T> : System.Buffers.IRetainable, Syste
         public abstract void Retain();
         protected internal abstract bool TryGetArray(out System.ArraySegment<T> arraySegment);
     }
+    public readonly partial struct ReadOnlyBuffer<T>
+    {
+        private readonly object _dummy;
+        public static readonly System.Buffers.ReadOnlyBuffer<T> Empty;
+        public ReadOnlyBuffer(System.Buffers.IMemoryList<T> startSegment, int offset, System.Buffers.IMemoryList<T> endSegment, int endIndex) { throw null; }
```

Comment:
the int parameter names should be consistent, e.g. startSegment, startIndex, endSegment, endIndex.